pkg/apis/kops/channel.go

```diff
@@ -248,6 +248,7 @@ const CloudProviderBareMetal CloudProviderID = "baremetal"
 const CloudProviderGCE CloudProviderID = "gce"
 const CloudProviderDO CloudProviderID = "digitalocean"
 const CloudProviderVSphere CloudProviderID = "vsphere"
+const CloudProviderExternal CloudProviderID = "external"
```

Comment:
Is this an external cloud provider, or is the aws cloud provider being used but in an external manner?  The cloud provider to me is aws / gce / vsphere, etc.

Reply:
Agree with Chris, the `CloudProviderID` type is used for kops to identify the true cloud provider name so having `external` here would be confusing.